services/pull/pull.go

```diff
@@ -499,6 +499,9 @@ func CloseRepoBranchesPulls(doer *models.User, repo *models.Repository) error {
 		}
 
 		for _, pr := range prs {
+			if pr.BaseRepoID == repo.ID {
```

Comment:
Can you add a comment describing which case this covers, it took me a moment to grasp
I.e. `We want to close close PRs targeting this repo, but avoid closing PRs originating from this repo`.

Reply:
```suggestion
			// If the base repository for this pr is this repository there is no need to close it
			// as it is going to be deleted anyway
			if pr.BaseRepoID == repo.ID {
```